### PR TITLE
data: ShailavMalik

### PR DESCRIPTION
**Pull request by ShailavMalik**
Added my profile

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4475"></a>

### CODE REVIEW

File: data/ShailavMalik.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+    "name": "Shailav Malik",
+    "displayStatsPublic": true,
+    "type": "personal",
+    "bio": "Coding Enthusiast | Frontend development",
+    "avatar": "https://github.com/ShailavMalik.png",
```
Comment (Krishnapro) [inline data/ShailavMalik.json:6]: @ShailavMalik remove the avatar it will be taken from your GitHub

**Review by Krishnapro — CHANGES_REQUESTED** [review]

File: data/ShailavMalik.json (hunk 1 above)
Comment (ShailavMalik) [reply data/ShailavMalik.json:6]: OK

File: data/ShailavMalik.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    "links": [
+        {
+            "name": "Follow on Instagram for coding/tech content",
+            "url": "https://www.instagram.com/shailavmalik1/",
+            "icon": "SiInstagram"
+        },
+        {
+            "name": "Follow me on LinkedIN",
```
Comment (priyankarpal) [inline data/ShailavMalik.json:14]:
``` suggestion
            "name": "Follow me on LinkedIn",
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: data/ShailavMalik.json (hunk 2 above)
Comment (priyankarpal) [reply data/ShailavMalik.json:14]: Just change the spelling

**Review by priyankarpal — APPROVED** [review]
LGTM ⚡

File: data/ShailavMalik.json (hunk 1 above)
Comment (ShailavMalik) [reply data/ShailavMalik.json]: Made the changes

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks for your Contribution! 🎉

**Review by Krishnapro — APPROVED** [review]
@ShailavMalik LGTM

**Review by priyankarpal — APPROVED** [review]
LGTM 👍🏻

**Review by iamthecloverly — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Pull request merged**